oeps/oep-0010.rst

```diff
@@ -0,0 +1,202 @@
+=========================
+OEP-10: Open edX Releases
+=========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-10 <oep-0010>`                  |
++---------------+-------------------------------------------+
+| Title         | Open edX Releases                         |
++---------------+-------------------------------------------+
+| Last Modified | 2016-10-14                                |
++---------------+-------------------------------------------+
+| Author        | Ned Batchelder, ned@edx.org               |
++---------------+-------------------------------------------+
+| Arbiter       | Edward Fagin                              |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Process                                   |
++---------------+-------------------------------------------+
+| Created       | 2016-10-14                                |
++---------------+-------------------------------------------+
+| `Resolution`  | <links to any discussions where the final |
+|               | status was decided>                       |
++---------------+-------------------------------------------+
+
+
+..
+    - Expectations for component owners
+
+
+Abstract
+========
+
+Open edX is packaged occasionally into releases. This document details the
+process.
+
+
+Motivation
+==========
+
+Open edX releases happen infrequently, every four to six months. Edx engineers
```

Comment:
We are definitely discussing working to a more specific schedule. Frankly, I'm not sure we have the organizational maturity to do it, though.